recipe/conda_build_config.yaml

```diff
@@ -2,3 +2,6 @@ dnnl_cpu_runtime:
   - tbb  # [not (ppc64le or aarch64)]
   - omp
   - threadpool # [not (ppc64le or aarch64 or arm64)]
+dnnl_gpu_runtime:
+  - ocl   # [linux64 or win64]
+  - none  # [not (linux64 or win64)]
```

Comment:
I am not sure this is the correct check. If there is a linux64 machine with nvidia GPU then OCL is not supported.

I would highly recommend to enable SYCL runtime instead of OCL, because this is the main GPU runtime in oneDNN. Frameowkrs like PyTorch and TensorFlow are enabled using SYCL:
- https://github.com/intel/intel-extension-for-tensorflow 
- https://github.com/intel/intel-extension-for-pytorch

Also oneDNN has community-based support for Nvidia (cudnn & cublas based) and AMD (miopen based) GPUs:
- https://github.com/oneapi-src/oneDNN/tree/master/src/gpu/nvidia 
- https://github.com/oneapi-src/oneDNN/tree/master/src/gpu/amd

Is there any particular reason for enabling OCL?

Reply:
@isuruf Could you please take a look at #53 ?